packages/meditrak-server/src/app/addRoutesToApp.js

```diff
@@ -152,9 +152,9 @@ export function addRoutesToApp(app) {
    */
   app.get('(/v[0-9]+)?/changes/count', countChanges);
   app.get('(/v[0-9]+)/export/surveyResponses', exportSurveyResponses);
-  app.get('(/v[0-9]+)/export/surveyResponse/:surveyResponseId', exportSurveyResponses);
+  app.get('(/v[0-9]+)/export/surveyResponses/:surveyResponseId', exportSurveyResponses);
```

Comment:
We may use this endpoint in web-config-server as part of the tupaia.org raw data downloads system - could you check?

Reply:
Did a grep through web-config-server and all references use the batch `/export/surveyResponses` endpoint (which has always been plural) not the individual `/export/surveyResponses/:id` endpoint

Reply:
Great! Thanks for checking